### PR TITLE
feat(route): Add Ash Maurya's blog

### PR DESCRIPTION
**Pull request by james-tindal**
## 完整路由地址 / Example for the proposed route(s)

```routes
/ash-maurya
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [ ] CN
  - [x] EN
- [x] 全文获取 fulltext
  - [x] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note
The blog is [here](https://blog.leanstack.com/).

### CODE REVIEW

**Comment by TonyRL** [comment]
Missing `maintainer.js`, `radar.js` and documentation. Also, would you run `npm run format` which should auto-fix some/all of the lint issues?

**Comment by james-tindal** [comment]
Thanks @TonyRL. I did what you suggested. Not 100% sure maintainer.js and radar.js are correct though.

File: lib/v2/blogs/love-the-problem/radar.js
Comment (TonyRL) [inline lib/v2/blogs/love-the-problem/radar.js:2]:
```suggestion
    'blog.leanstack.com': {
```

File: docs/blog.md
Comment (TonyRL) [inline docs/blog.md:79]:
```suggestion
<RouteEn author="james-tindal" example="/blogs/love-the-problem" path="/blogs/love-the-problem"/>
```
Apart from the changes, you should add this documentation in docs/en/blog.md instead as it’s in English.

File: docs/blog.md
Comment (TonyRL) [inline docs/blog.md:77]:
```suggestion
### Ash Maurya's blog
```

File: lib/v2/blogs/love-the-problem/index.js
Comment (TonyRL) [inline lib/v2/blogs/love-the-problem/index.js:39]:
```suggestion
                                description: (content('.post-full-image').html() ?? '') + content('.post-content').html(),
```

File: lib/v2/blogs/love-the-problem/maintainer.js
Comment (TonyRL) [inline lib/v2/blogs/love-the-problem/maintainer.js:2]:
```suggestion
    '/love-the-problem': ['james-tindal'],
```

File: lib/v2/blogs/love-the-problem/radar.js
Comment (TonyRL) [inline lib/v2/blogs/love-the-problem/radar.js:6]: I think `Ash Maurya's blog` is better.

File: lib/v2/blogs/love-the-problem/radar.js
Comment (TonyRL) [inline lib/v2/blogs/love-the-problem/radar.js:7]:
```suggestion
                docs: 'https://docs.rsshub.app/en/blog.html#love-the-problem',
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/blogs/love-the-problem/maintainer.js
Comment (TonyRL) [reply lib/v2/blogs/love-the-problem/maintainer.js:2]: Should be `lib/v2/blogs/maintainer.js`

File: lib/v2/blogs/love-the-problem/radar.js
Comment (james-tindal) [reply lib/v2/blogs/love-the-problem/radar.js:6]: I was considering this. Should we change the name everywhere?

Comment (james-tindal) [reply lib/v2/blogs/love-the-problem/radar.js:6]: route: /blogs/ash-maurya ?

File: docs/blog.md
Comment (james-tindal) [reply docs/blog.md]: This is blocked by #8346

**Comment by NeverBehave** [comment]
Once you guys are ready, request a review from me

**Comment by james-tindal** [comment]
@NeverBehave 
All looks good to me!

**Review by DIYgod — APPROVED** [review]

**Pull request merged**
